<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fourthPage</class>
 <widget class="QWidget" name="fourthPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_open_window">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Refresh_Background">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh Background</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_polygonzug">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Polygonzug</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_one_node">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_remove_last_node">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cut_polygonzug">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Cut
Polygonzug</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start_set_position_of_laser">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Start
Set Position 
of Laser</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_done_set_position_of_laser">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>121</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Done 
Set Position 
of Laser</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_laserX">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox_laserY">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_vel_PolygonZug">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>380</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ScreenSootPosition width and height</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>306</width>
     <height>134</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_up">
          <property name="text">
           <string>up</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_left">
          <property name="text">
           <string>left</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_right">
          <property name="text">
           <string>right</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_down">
          <property name="text">
           <string>down</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>w</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>h</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSpinBox" name="spinBox_screenShot_x"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_screenShot_y"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_screenShot_w"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_screenShot_h"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>StepSize</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_stepSize"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>pushButton_open_window</zorder>
  <zorder>pushButton_Refresh_Background</zorder>
  <zorder>checkBox_polygonzug</zorder>
  <zorder>pushButton_add_one_node</zorder>
  <zorder>pushButton_remove_last_node</zorder>
  <zorder>pushButton_cut_polygonzug</zorder>
  <zorder>pushButton_start_set_position_of_laser</zorder>
  <zorder>pushButton_done_set_position_of_laser</zorder>
  <zorder>spinBox_laserX</zorder>
  <zorder>spinBox_laserY</zorder>
  <zorder>spinBox_vel_PolygonZug</zorder>
  <zorder>pushButton_left</zorder>
  <zorder>pushButton_up</zorder>
  <zorder>pushButton_down</zorder>
  <zorder>pushButton_right</zorder>
  <zorder>spinBox_screenShot_y</zorder>
  <zorder>spinBox_screenShot_x</zorder>
  <zorder>label</zorder>
  <zorder>spinBox_screenShot_w</zorder>
  <zorder>spinBox_screenShot_h</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>spinBox_stepSize</zorder>
  <zorder>label_6</zorder>
  <zorder>spinBox_screenShot_x</zorder>
  <zorder>spinBox_screenShot_h</zorder>
  <zorder>spinBox_screenShot_y</zorder>
  <zorder>spinBox_screenShot_w</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
